components/prism-hoon.js

```diff
@@ -0,0 +1,30 @@
+Prism.languages.hoon = {
+	constant: /(?:%\.n)|(?:%\.y)|(?:%[\w-]+\w?)/,
+	comment: {
+		pattern: /::.*/,
+		lookahead: true,
+		greedy: true
+	},
+	'function-name': /(?:\+[-+] {2})?(?:[a-z]([a-z0-9-]*[a-z0-9])?)/,
+	'class-name': [
+		{
+			pattern: /@(?:[a-z])?(?:[A-Za-z0-9-]*[A-Za-z0-9])?/,
```

Comment:
This can be simplified.

Also, I don't think it's right that just `@` is a class name or is it?

```suggestion
			pattern: /@(?:[A-Za-z0-9-]*[A-Za-z0-9])?/,
```

Reply:
Just `@` is an atom, the most basic data structure in Nock, and is therefore valid to reference along any other aura (or typed atom). (As for why class name — placement of Urbit-specific terms into syntax highligher-adjacent terms I believe I took from our vim definition, but I can't recall at the moment.)